docs/tutorials/lut.rst

```diff
@@ -59,35 +59,35 @@ In all cases we will make use of the :mod:`QMigrate.io` module, so let's import
 
 ::
 
-    import QMigrate.io as qio
+    from QMigrate.io import read_stations, read_vmodel
```

Comment:
Hasn't been changed to read_stations yet? But if you're going to change this imminently then all good. (But note it needs changing in the other examples.)

Reply:
Pushed this up to refactor_magnitudes (e89c3b0)